app/models/container.rb

```diff
@@ -2,6 +2,7 @@ class Container < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
   include ArchivedMixin
```

Comment:
@zeari well, merging them is one option.

But I was saying that since `ArchivedPurgingMixin` includes `ArchivedMixin`, there is no reason to keep the `include ArchivedMixin` in these files.